runner/docker/Dockerfile.live-base-comfyui

```diff
@@ -81,6 +81,5 @@ RUN rm -rf /workspace/ComfyUI/output && ln -s /models/ComfyUI--output /workspace
 RUN echo "source /workspace/miniconda3/etc/profile.d/conda.sh && conda activate comfystream" >> /conda_activate.sh && \
     chmod +x /conda_activate.sh
 ENV BASH_ENV=/conda_activate.sh
-SHELL ["/bin/bash", "-c"]
```

Comment:
neat, love not needing this lol

Reply:
Yes, but there is still an opportunity to remove https://github.com/livepeer/ai-runner/pull/479/files?w=0#diff-443cdad78f488f139f012984eac0c36f930e5ceb4ce1dd436f19c865e7ddc639R81-R83. The process that is starting ComfyUI is infer.py. This can likely be solved with a change to start infer.py in the comfystream environment `cmd = ["conda", "run", "-n", "comfystream", "python", str(self.infer_script_path)]` https://github.com/livepeer/ai-runner/blob/3de85765f05314b6f86a8633bc2a548ceb94ff7f/runner/app/pipelines/live_video_to_video.py#L109-L111

Reply:
I was able to solve this by starting infer.py with the comfystream conda env https://github.com/livepeer/ai-runner/pull/479/commits/2e046bb50ab54d69806f1958c19c6d1ac445e759